src/chains/blast/core/BlastMgvView.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {MgvView} from "../../../core/MgvView.sol";
+import {IBlastPoints} from "../interfaces/IBlastPoints.sol";
+
+/// @title BlastMgvView
+/// @notice A contract that inherits MgvView
+/// @dev This contract is used to get the points admin for blast
+contract BlastMgvView is MgvView, IBlastPoints {
+  function blastPointsAdmin() external view override returns (address) {
+    return _governance;
+  }
```

Comment:
Could we put this function in `BlastMangrove` or will this make the contract too big?

If we could, I think we could avoid this file and `BlastMgvAppendix` + we wouldn't need to modify `Mangrove.sol`.